lib/smart_answer/calculators/maternity_paternity_calculator.rb

```diff
@@ -226,11 +226,14 @@ def total_statutory_pay
     def paydates_and_pay
       paydates = send(:"paydates_#{pay_method}")
       [].tap do |ary|
+        last_paydate = pay_start_date
         paydates.each_with_index do |paydate, index|
```

Comment:
do we need `each_with_index` anymore? - it looks like we're no longer using `index`

Reply:
Nope, removed